lib/roadie/rails/asset_pipeline_provider.rb

```diff
@@ -18,7 +18,13 @@ def find_stylesheet(name)
 
       private
       def normalize_asset_name(href)
-        remove_asset_prefix href.split('?').first
+        res = remove_asset_prefix href.split('?').first
+        res = remove_asset_digest res
+        res
```

Comment:
You don't need to reassign the variable here to just return it.

```ruby
  remove_asset_digest res
end
```

would suffice.